docs/OV_Runtime_UG/ov_transformations.md

```diff
@@ -24,7 +24,6 @@ Template for FunctionPass transformation class
 
 Using `ov::pass::ModelPass`, you need to override the `run_on_model` method where you will write the transformation code.
```

Comment:
`Template for FunctionPass` => `Template for ov::pass::ModelPass`

Reply:
remove ngraph everywhere in this file including code snippets, function to model